app/src/pages/inside/launchesPage/launcheSuiteGrid/itemInfo/durationBlock/durationBlock.jsx

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright 2017 EPAM Systems
+ *
+ *
+ * This file is part of EPAM Report Portal.
+ * https://github.com/reportportal/service-ui
+ *
+ * Report Portal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Report Portal is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Report Portal.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { Component } from 'react';
+import classNames from 'classnames/bind';
+import Parser from 'html-react-parser';
+import { getDuration, dateFormat, approximateTimeFormat } from 'common/utils';
+import { injectIntl, intlShape, defineMessages } from 'react-intl';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import ClockIcon from './img/clock-icon-inline.svg';
+import InProgressGif from './img/item-in-progress.gif';
+
+import styles from './durationBlock.scss';
+
+const cx = classNames.bind(styles);
+const messages = defineMessages({
```

Comment:
such messages should be defined in high level

Reply:
TBD